mlfinlab/ensemble/sb_bagging.py

```diff
@@ -0,0 +1,659 @@
+"""
+Implementation of Sequentially Bootstrapped Bagging Classifier using sklearn's library as base class
+"""
+import numbers
+import itertools
+from warnings import warn
+from abc import ABCMeta, abstractmethod
+import pandas as pd
+import numpy as np
+
+from sklearn.tree import DecisionTreeClassifier, DecisionTreeRegressor
+from sklearn.metrics import accuracy_score, r2_score
+from sklearn.ensemble.bagging import BaseBagging, BaggingClassifier, BaggingRegressor
+from sklearn.base import ClassifierMixin, RegressorMixin
+from sklearn.ensemble.base import _partition_estimators
+from sklearn.utils.random import sample_without_replacement
+from sklearn.utils.validation import has_fit_parameter
+from sklearn.utils import check_random_state, indices_to_mask, check_array, check_consistent_length, check_X_y
+from sklearn.utils._joblib import Parallel, delayed
+
+from mlfinlab.sampling.bootstrapping import seq_bootstrap, get_ind_matrix
+
+MAX_INT = np.iinfo(np.int32).max
+
+
+# pylint: disable=R0901
```

Comment:
Are these errors specific to sklearns implementation? I prefer swapping the code with the name so its more clear what they are disabling.

Example: `# pylint: disable=R0901 -> # pylint: disable=too-many-ancestors`

Reply:
Sure, let me change that